### ISSUE TITLE
Insert/Edit Link URL dropdown items not announced by screen readers

### ISSUE THREAD

**Issue by stefandobre** [tinymce/tinymce#8728]
<img width="490" alt="url-dropdown" src="https://github.com/tinymce/tinymce/assets/20504443/85fc4ae4-ce1e-41ac-9a9f-d205ad7c9c9c">

The URL dropdown items \<top\> and \<bottom\> are not getting announced when selected. Tested with Voiceover on MacOS and Jaws 2023 on Windows.

It appears the `aria-selected` attribute is not getting set.

The "Open Link in..." select list at the bottom of the dialog works as expected.

**Comment by TheSpyder** [comment]
We believe we improved this in #8593, it will be released with TinyMCE 6.5. You can try it right now on the `dev` fiddle branch:
https://fiddle.tiny.cloud/X7iaab

Please let us know if this helps.
